sklearn/utils/_indexing.py

```diff
@@ -520,10 +531,21 @@ def resample(*arrays, replace=True, n_samples=None, random_state=None, stratify=
         )
 
     check_consistent_length(*arrays)
```

Comment:
```suggestion
    check_consistent_length(*arrays)

```

Reply:
We still need to commit this suggestion to re-add the missing newline.